src/index.js

```diff
@@ -1,35 +1,73 @@
-const core = require('@actions/core');
-const github = require('@actions/github');
+const core = require("@actions/core");
+const github = require("@actions/github");
 
 async function main() {
-  const { eventName, sha, ref, repo: { owner, repo }, payload } = github.context;
+  const {
+    sha,
+    ref,
+    repo: { owner, repo },
+    payload
+  } = github.context;
+
+  // GITHUB_RUN_ID is provided by Github
+  const run_id = process.env.GITHUB_RUN_ID;
```

Comment:
Although GitHub uses `snake_case` in its REST API, we still only use `camelCase` for naming variables and functions.
```suggestion
  const runId = process.env.GITHUB_RUN_ID;
```